### PR TITLE
Base React App Upload

### PR DESCRIPTION
**Pull request by zacharygilbert437**

### CODE REVIEW

**Comment by oresttuz** [comment]
Looks good

**Review by oresttuz — COMMENTED** [review]
Nice README

**Review by oresttuz — APPROVED** [review]
Good to go

**Review by dylanlee5 — APPROVED** [review]
looks good to me

File: README.md [hunk 1]
```diff
@@ -511,3 +511,5 @@ Return:
     User
 }
 ```
+
+Zach can change files!
```
Comment (ChrisGambrell) [inline README.md:515]: Testing using “request change” feature. 

Maybe add your name under “Authors”?

**Review by ChrisGambrell — CHANGES_REQUESTED** [review]

**Comment by ChrisGambrell** [comment]
This also should be converted to a different PR because we need to merge it into the `secondary` branch, not `main`

**Comment by ChrisGambrell** [comment]
Accidentally opened issue above ^

**Pull request closed without merging**